var/spack/repos/builtin/packages/py-scipy/package.py

```diff
@@ -111,9 +111,17 @@ class PyScipy(PythonPackage):
     # and setup_build_environment() from numpy in the scipy spec.
     depends_on("blas")
     depends_on("lapack")
-    # https://github.com/scipy/scipy/wiki/Dropping-support-for-Accelerate
+
+    # https://docs.scipy.org/doc/scipy/dev/toolchain.html#other-libraries
+    depends_on("lapack@3.7.1:", when="@1.9:")
     depends_on("lapack@3.4.1:", when="@1.2:")
 
+    # https://docs.scipy.org/doc/scipy/dev/toolchain.html#compilers
+    conflicts("%gcc@:7", when="@1.10:")
+    conflicts("%gcc@:4.7", when="@:1.9")
+    conflicts("%apple-clang@:9", when="@1.10:")
+    conflicts("%msvc@:18", when="@1.10:")
```

Comment:
Maybe `mscv@:19.10`? The `meson.build` in 1.10.x and main contains:
```meson
elif cc.get_id() == 'msvc'
  if not cc.version().version_compare('>=19.20')
    error('SciPy requires at least vc142 (default with Visual Studio 2019) ' + \
          'when building with MSVC')
  endif
endif
```

Good catch on all these!